src/vue-parallax-js.js

```diff
@@ -1,4 +1,12 @@
 // @flow
+if (typeof window === 'undefined') {
+  global.window = null
+}
+
+if (typeof document === 'undefined') {
```

Comment:
What's the point for mutating the global variable to null when they're undefined?